h5py/_hl/files.py

```diff
@@ -400,12 +400,21 @@ def __init__(self, name, mode='r', driver=None, libver=None, userblock_size=None
             recommended), 'core', 'sec2', 'direct', 'stdio', 'mpio', 'ros3'.
         libver
             Library version bounds.  Supported values: 'earliest', 'v108',
-            'v110', 'v112'  and 'latest'.
+            'v110', 'v112', 'v114' and 'latest'.
```

Comment:
You can add `v200` here, too, as it will soon be available.

Reply:
done with @tacaswell's suggestion !